docs/index.rst

```diff
@@ -9,9 +9,7 @@ Introduction
 
 The goal of skorch is to make it possible to use PyTorch_ with
 sklearn_. This is achieved by providing a wrapper around
-PyTorch that has an sklearn interface. In that sense,
-skorch is the spiritual successor to nolearn_, but instead of
-using Lasagne and Theano, it uses PyTorch.
```

Comment:
`Lasagne` were good times :)

Reply:
Yes, Theano and Lasagne were great, but history will only remember keras. Maybe we should sit down one day and write a history for posterity :D